packit_service/service/events.py

```diff
@@ -418,6 +396,4 @@ def get_package_config(self):
         return package_config
 
     def get_project(self) -> GitProject:
```

Comment:
could we put this method to `AbstractGithubEvent` as it is the same for all of the Github Events?